hi-ml-histopathology/src/histopathology/configs/SSL/CRCK_SimCLRContainer.py

```diff
@@ -33,22 +32,22 @@ class CRCK_SimCLR(HistoSSLContainer):
                                                TcgaCrck_TilesDatasetWithReturnIndex})
 
     def __init__(self, **kwargs: Any) -> None:
-        if not is_running_in_azure_ml():
-            is_debug_model = True
-            num_workers = 0
-            max_epochs = 2
+        # if not running in Azure ML, you may want to override certain properties the command line, such as:
+        # --is_debug_model = True
+        # --num_workers = 0
+        # --max_epochs = 2
 
         super().__init__(ssl_training_dataset_name=SSLDatasetNameHiml.TCGA_CRCK,
                          linear_head_dataset_name=SSLDatasetNameHiml.TCGA_CRCK,
                          azure_datasets=["TCGA-CRCk"],
                          random_seed=1,
-                         num_workers=num_workers,
-                         is_debug_model=is_debug_model,
+                         num_workers=8,
+                         is_debug_model=False,
                          model_checkpoint_save_interval=50,
                          model_checkpoints_save_last_k=3,
                          model_monitor_metric='ssl_online_evaluator/val/AreaUnderRocCurve',
                          model_monitor_mode='max',
-                         max_epochs=max_epochs,
+                         max_epochs=50,
```

Comment:
are these changes still remnants of your playing with the code or itentional?

Reply:
Intentional - previously the values of max_epochs, num_workers and is_debug were only getting set when not running in AML. I commented out these variables and set default values for these properties, the same way we do in the other configs